LayoutTests/imported/w3c/web-platform-tests/css/cssom/serialize-values-expected.txt

```diff
@@ -326,10 +326,10 @@ PASS content: attr(foo-bar)
 PASS content: attr(foo_bar)
 FAIL content: attr(|bar) assert_equals: content raw inline style declaration expected "attr(bar)" but got ""
 FAIL content: attr( |bar ) assert_equals: content raw inline style declaration expected "attr(bar)" but got ""
-FAIL content: attr(foo-bar, "fallback") assert_equals: content raw inline style declaration expected "attr(foo-bar, \"fallback\")" but got ""
-FAIL content: attr(foo_bar, "fallback") assert_equals: content raw inline style declaration expected "attr(foo_bar, \"fallback\")" but got ""
+PASS content: attr(foo-bar, "fallback")
+PASS content: attr(foo_bar, "fallback")
 FAIL content: attr(|bar, "fallback") assert_equals: content raw inline style declaration expected "attr(bar, \"fallback\")" but got ""
-FAIL content: attr(foo, "") assert_equals: content raw inline style declaration expected "attr(foo)" but got ""
+FAIL content: attr(foo, "") assert_equals: content raw inline style declaration expected "attr(foo)" but got "attr(foo, \"\")"
```

Comment:
Can you run this test with `--reset-results` again? It should be passing with the change I suggested